ops/ansible/playbooks-ccs/vars/test/group_vars/all/env_specific.yml

```diff
@@ -3,12 +3,6 @@
 # Version of the ETL Jar to deploy 
 data_pipeline_version: '0.1.0-SNAPSHOT'
 
-# Bucket where deployment artifacts are located. 
-s3_bucket_deployment: 'bfd-mgmt-artifacts-577373831711'
```

Comment:
The test vault also has this value listed as `vault_s3_bucket_deployment`, but none of the other envs have it. Any reason for that?

Reply:
It looks to have existed there before this PR, per the master branch.  Lack of diff in vault is rough  :/